### PR TITLE
Form title 28-8832 -> 25-8832

### PR DESCRIPTION
**Pull request by DLarson-Oddball**
## Summary

Updated form name 28-8832 to 25-8832. 
Related PR: https://github.com/department-of-veterans-affairs/content-build/pull/1670

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/62663

## Testing done

- Previously, the page title and content included references to form 28-8832. There was also a comparison to form 25-8832 noting they are equivalent forms. 
- The page title and content now references form 25-8832. The comparison was also updated to form 28-8832.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |   image  |  image |
| Desktop |  image |  image |

## What areas of the site does it impact?

This PR will not impact pages outside of form 25-8832

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable). N/A
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution N/A
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable) N/A

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

### CODE REVIEW

**Review by fred-khoury-oddball — APPROVED** [review]

File: src/applications/vre/28-8832/manifest.json
Comment (Mottie) [inline src/applications/vre/28-8832/manifest.json:5]: If you ever need to update this path or entryName, it might be better to duplicate this entry with the new `entryName` & `rootUrl` first, then get devops to add a redirect to the new path. Then, remove this older entry.

**Review by Mottie — APPROVED** [review]

File: script/github-actions/daily-product-scan/tests/mocks/applications/app-2/app-2b/containers/WizardContainer.jsx
Comment (Mottie) [inline script/github-actions/daily-product-scan/tests/mocks/applications/app-2/app-2b/containers/WizardContainer.jsx:31]: If you remove this change, I don't think you'll need QA standards to approve this PR. I believe this "daily-product-scan" folder gets updated automatically from the content build repo `registry.json` - which would also require the same changes as the `manifest.json` file.

**Review by CBonade — APPROVED** [review]

File: script/github-actions/daily-product-scan/tests/mocks/applications/app-2/app-2b/containers/WizardContainer.jsx
Comment (fred-khoury-oddball) [reply script/github-actions/daily-product-scan/tests/mocks/applications/app-2/app-2b/containers/WizardContainer.jsx:31]: Thank you @Mottie this is very helpful. To clarify my understanding we can remove the above change and the changes in [this PR](https://github.com/department-of-veterans-affairs/content-build/pull/1670/files) should make their way back to here? Does this sounds correct?

**Comment by DLarson-Oddball** [comment]
@Mottie @CBonade I refactored my approach to adding a new entry for the form name change in content-build, so I wanted to re-request your review.

After merge of https://github.com/department-of-veterans-affairs/content-build/pull/1670, I will merge this pr along with a devops PR to create a redirect.

**Comment by Mottie** [comment]
I made a change to devops recently, and just so you're aware, their deploy cycle isn't daily. So make sure to contact them before trying to synchronize merges.

**Comment by DLarson-Oddball** [comment]
@Mottie or @CBonade Hoping you have time to give this another pass. This changes the approach by creating a new app with name 25-8832. Content changes are hosted in this app. 

Approach wise, I renamed the original app from 28-8832 to 25-8832 and created a new app named 28-8832. This will preserve git history in an upcoming PR which will remove 28-8832.

Review wise, The new files are copies of those in 28-8832 with some small copy changes.

File: src/applications/vre/25-8832/config/form.js
Comment (Mottie) [inline src/applications/vre/25-8832/config/form.js:26]: Are you planning on updating this tracking prefix?

File: src/applications/vre/25-8832/config/form.js
Comment (Mottie) [inline src/applications/vre/25-8832/config/form.js:30]: `VA_FORM_IDS` should also be updated

File: src/applications/vre/25-8832/config/form.js
Comment (Mottie) [inline src/applications/vre/25-8832/config/form.js:1]: Probably won't need to update `vets-json-schema` any time soon

File: src/applications/vre/25-8832/containers/ConfirmationPage.jsx
Comment (Mottie) [inline src/applications/vre/25-8832/containers/ConfirmationPage.jsx:30]: Update

File: src/applications/vre/25-8832/containers/WizardContainer.jsx
Comment (Mottie) [inline src/applications/vre/25-8832/containers/WizardContainer.jsx:31]: Update ID here

File: src/applications/vre/25-8832/containers/IntroductionPage.jsx
Comment (Mottie) [inline src/applications/vre/25-8832/containers/IntroductionPage.jsx:23]: Update ID here

File: src/applications/vre/25-8832/tests/e2e/formDataSets/veteran-workflow-test.json
Comment (Mottie) [inline src/applications/vre/25-8832/tests/e2e/formDataSets/veteran-workflow-test.json:15]: LOL maybe we should replace the previous developer's email in the data

File: src/applications/vre/28-8832/containers/WizardContainer.jsx
Comment (Mottie) [inline src/applications/vre/28-8832/containers/WizardContainer.jsx:28]: Update here?

**Review by Mottie — APPROVED** [review]

File: src/applications/vre/25-8832/config/form.js
Comment (DLarson-Oddball) [reply src/applications/vre/25-8832/config/form.js:26]: Not currently unless you feel can include this without impacting other teams. I'm not sure what it's used for.

File: src/applications/vre/25-8832/containers/ConfirmationPage.jsx
Comment (DLarson-Oddball) [reply src/applications/vre/25-8832/containers/ConfirmationPage.jsx:30]: Actually, this is correct. There is a specific note on the page indicating the forms are equivalent. Since titles and breadcrumbs indicate this is form 25-8832, this notes the form equivalence.

File: src/applications/vre/25-8832/containers/WizardContainer.jsx
Comment (DLarson-Oddball) [reply src/applications/vre/25-8832/containers/WizardContainer.jsx:31]: Same as above

File: src/applications/vre/25-8832/containers/IntroductionPage.jsx
Comment (DLarson-Oddball) [reply src/applications/vre/25-8832/containers/IntroductionPage.jsx:23]: Same as above

File: src/applications/vre/28-8832/containers/WizardContainer.jsx
Comment (DLarson-Oddball) [reply src/applications/vre/28-8832/containers/WizardContainer.jsx:28]: Keeping the content the same as this app is 28-8832, but fixed spacing.

File: src/applications/vre/25-8832/config/form.js
Comment (DLarson-Oddball) [reply src/applications/vre/25-8832/config/form.js:1]: Can you expand what you mean by this?

File: src/applications/vre/25-8832/config/form.js
Comment (DLarson-Oddball) [reply src/applications/vre/25-8832/config/form.js:30]: Agreed this should be updated. I will have a follow up PR that removes the old app (28-8832) once the redirect is in place. Since updating the form id requires approval from other teams, I will just include it in that PR.

**Pull request merged**
